libs/db/fsdb.go

```diff
@@ -18,6 +18,14 @@ const (
 	dirPerm = os.FileMode(0700)
 )
 
+// PermissionsChangedErr occurs if the file permission have changed since the file was created.
+type PermissionsChangedErr string
```

Comment:
Prefix over suffix as above.

Is there a reason that both packages need to export their own version of the same error, can those be consolidated?

Reply:
`libs/common` does have errors, but comes with a lot of other things. I'd say a central `errors` package would be good, maybe for now under libs.

Reply:
good point. done.